### ISSUE TITLE
Date Picker - when added necessity indicator, alignment breaks

### ISSUE THREAD

**Issue by AnkitaBagale** [razorpay/blade#2196]
when added necessity indicator, alignment of arrow breaks
Also end date shows * on top.

<img width="806" alt="image" src="https://github.com/razorpay/blade/assets/69760792/3da1df28-b628-4c69-ac19-7944a5ab7c6a">

```
    <DatePicker
      label={{
        start: 'Validity',
      }}
      labelPosition="left"
      onChange={(value) => {
        console.log('onChange', value);
      }}
      presets={getPresets()}
      selectionType="range"
      isRequired
      necessityIndicator="required"
    />
```

**Comment by anuraghazra** [comment]
Will fix this.

**Comment by anuraghazra** [comment]
This is fixed and will be released in v11.19.1 in a few minutes.
